invenio_jobs/errors.py

```diff
@@ -8,10 +8,10 @@
 """Errors for logging."""
 
 
-class TaskExecutionError(Exception):
+class TaskExecutionPartialError(Exception):
```

Comment:
maybe it would be better to add a new error class and mark the old one as deprecated. renaming would potentially break others code.

Reply:
yes, good idea

Reply:
Good point! Done